app/javascript/components/profile/metrics.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <div class="profile-metrics">
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Expected indentation of 2 spaces but found 4 spaces    vue/html-indent